src/components/icons.js

```diff
@@ -60,14 +60,19 @@ export const centeredSpinner = ({ size = 48, ...props } = {}) => spinner(_.merge
   }
 }, props))
 
-export const profilePic = ({ size, style, ...props } = {}) => img({
-  alt: 'Google profile image',
-  src: getUser().imageUrl,
-  height: size, width: size,
-  style: { borderRadius: '100%', ...style },
-  referrerPolicy: 'no-referrer',
-  ...props
-})
+export const profilePic = ({ size, style, ...props } = {}) => {
+  // Note Azure logins don't currently have an imageUrl, so don't render anything.
+  // See TOAZ-147 to support this in the future.
+  const imageUrl = getUser().imageUrl
+  return imageUrl && img({
```

Comment:
```suggestion
  return !!imageUrl && img({
```
Since this is returning something that is going to be treated as a component to render, the check should be coerced to a boolean. If `imageUrl` is `undefined`, this will do what you expect. However, if it is `null` or 0, it will still be falsy, but I think React will end up rendering the string "null" or "0". Coercing it to a boolean will make it decisively `false` instead of relying on falsy.